### DISCUSSION TITLE
ImportError: Cannot Import 'AddChannel' from 'monai.transforms'

### DISCUSSION THREAD

**Discussion by delacruc** [Project-MONAI/MONAI#7865] [Q&A]
Hello MONAI Community,

I have encountered an issue while trying to import the AddChannel function from the monai.transforms module. The error message I receive is:

ImportError: cannot import name 'AddChannel' from 'monai.transforms' (/rds/homes/d/delacruc/.local/lib/python3.9/site-packages/monai/transforms/__init__.py)

**Comment by KumoLiu** [comment]
Hi @delacruc, `AddChannel` has been removed since 1.3.0, please try to use `EnsureChannelFirst` instead.
https://github.com/Project-MONAI/MONAI/pull/7019

Thanks.
